Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN if [ "$ENABLE_TDX" = "true" ]; then \
     fi
 
 # Final stage
-FROM --platform=$TARGETPLATFORM ubuntu:24.04
```

Comment:
No more platform support ?

Reply:
Setting platform to predefined `$TARGETPLATFORM ` in `FROM` is redundant as this is the default behaviour